02_getting_started.asciidoc

```diff
@@ -36,33 +36,48 @@ If users decide to outsource some functionality to a third party they usually gi
 
 Finally, those seeking simplicity and convenience, even at the expense of control and security, may choose a custodial Lightning wallet. This is the least technically challenging option, but it _undermines the trust model of cryptocurrency_ and should therefore be considered only as a stepping stone towards more control and self-reliance.
 
-In <<lnwallet-categories>> we see the three broad categories of lightning wallets and the relative degree of control they offer to the user.
+There are many ways wallets can be characterized or categorized. 
+From all features the two most important criteria are LN Node use and fund custody. Correspondingly the two most important questions to ask about a specific wallet are:
+
+- Does this Lightning wallet have a full LN Node or does it use a third-party LN Node?
+- Does this Lightning wallet store its own keys under user control (self-custody) or are the keys held be a third-party custodian?
+
+From these two questions we can derive four possible categories. We can place these four categories into a quadrant as seen in Table <<lnwallet-categories>>. 
+But remember that this is just one way of categorizing Lightning wallets.
 
 [[lnwallet-categories]]
-.Broad Categories of "Lightning Wallets"
+.Lightning Wallets Quadrant
 |===
-| Wallet Type          | LN Node        | Keystore/Custody | Technical Skill
-| Full Node & Wallet   | Full Node      | Non-Custodial    | High
-| Non-Custodial Wallet | 3rd-party node | Non-Custodial    | Medium
-| Custodial Wallet     | 3rd-party node | Custodial        | Low
+|                        | *Full LN Node*      | *3rd-party LN Node*    
+| *Self-Custody*         | Q1: High technical skill | Q2: Medium technical skills 
+| *Custodial*            | Q3: not a common wallet type | Q4: Low technical skills 
 |===
 
+Q3, quadrant 3, where a full LN node is used but the keys are held by a custodian is not common. A user who goes through the effort of running a Ligtning Node will most likely also put in the additional efffort to store his own keys. 
```

Comment:
True that it's not common, but it may become attractive in the future, as it would let a user worry about the operational aspects of their node, but then possibly delegate the access to the keys+signing to a 3rd party which may use primarily cold storage.

Reply:
I rephrased and worked your idea into the text. See Commit # 2.